docs/namespaces.md

```diff
@@ -1,36 +1,41 @@
 # Namespaces
 
-Nerve offers a rich set of predefined tools, organized in namespaces, that the agent can import [via the using directive](tasklets.md#tools). **If a specific model does not support tools calling natively, Nerve will use its own XML based format to enable tool calling.**
+Nerve offers a rich set of predefined tools, organized in namespaces, that the agent can import [with the using directive](tasklets.md#tools). **If a specific model does not natievly support tool calling, Nerve will use its own XML based format to enable tool calling.**
 
-The following is the list of namespaces available in Nerve, with the descriptive prompt that will be provided to the model.
+This page contains the list of namespaces available in Nerve, with the descriptive prompt that will be provided to the model.
 
 ## Memory
 
 Use these memory-related actions to store and retrieve any meaningful information obtained or interpreted from output generated by previous actions. Information worth storing as a memory includes anything potentially useful for or relevant to accomplishing your assigned task, goal, or plan. Memories should be deleted if they are no longer relevant to the current goal or if they contain erroneous info as determined by new info obtained through subsequent actions.
 
+<!-- I think the first sentence needs a tune-up. Perhaps splitting it into two would make it easier to follow? -->
 
 * To store a memory: `<save_memory key="my-note">put here the custom data you want to keep for later</save_memory>`
 * To delete a memory you previously stored given its key: `<delete_memory key="my-note"/>`
 
+<!-- Is there a reason the second bullet point says "given its key" but the first one doesn't? It looks like both code snippets mention the `key` - just checking! -->
```

Comment:
in the first case it's a "save a memory with any key" while in the second case the "given its key" implies that the key must exist (from a previously created memory)